standard/links/commons/links.lua

```diff
@@ -145,6 +145,7 @@ function Links.transform(links)
 		fanclub = links.fanclub,
 		gamersclub = links.gamersclub,
 		gamersclub2 = links.gamersclub2,
+		halodatahive = links.halodatahive,
```

Comment:
@hjpalpha Does this not need to be added to `_PREFIXES`?

Reply:
not necessarily
on halo it gets loaded via line 89 `_PREFIXES = Table.merge(_PREFIXES, CustomData.prefixes or {})`
and on other wikis it is not used

the question is if we should make it available in general
the customData stuff is orig intended for when wikis have to overwrite values (like e.g. with the faceit stuff where the prefixes have to be different between wikis)

Reply:
I think we should make it available then

Reply:
I'll be honest, my thinking was to follow how _aligulac_ was structured and implemented, as HDH only tracks halo stats. I'm not sure if that would have any weight with how it should be implemented though.

Reply:
I'll leave it to @hjpalpha's judgement

Reply:
imo move it to the commons module

Reply:
I am unsure what needs to be changed/ what edit's need to be made at this point. I believe you two have edit access to this branch?

Reply:
done